templates/mdd/task3_qarecs/templates.yaml

```diff
@@ -0,0 +1,84 @@
+dataset: mdd
+subset: task3_qarecs
+templates:
+  1614890b-362c-4ee8-850d-841cf511d169: !Template
+    id: 1614890b-362c-4ee8-850d-841cf511d169
+    jinja: "Speaker 1: {{dialogue_turns.utterance[0]}}
+
+
+      Speaker 2: {{dialogue_turns.utterance[1]}}
+
+
+      Speaker 1: {{dialogue_turns.utterance[2]}}
+
+
+      Speaker 2:|||{{dialogue_turns.utterance[3]}}"
+    name: next_utterance_4
+    reference: Given the first three dialogues,  generate next utterance.
+  3e5a19e5-aa33-467a-bcd2-f84d99f32759: !Template
+    id: 3e5a19e5-aa33-467a-bcd2-f84d99f32759
+    jinja: "{{dialogue_turns.utterance[0]}}|||{{dialogue_turns.utterance[1]}}"
+    name: recommend_movie_1
+    reference: Given likes, recommend a movie.
+  76888d6e-76fa-47db-a8b3-9980f082df51: !Template
+    id: 76888d6e-76fa-47db-a8b3-9980f082df51
+    jinja:
+      "{% if dialogue_turns.utterance|length==6 %}{{dialogue_turns.utterance[4]}}|||{{dialogue_turns.utterance[5]}}{%
+      else %}|||{% endif %}"
+    name: recommend_movie_2
+    reference:
+      Given a single preference, recommend a movie. Only works for dialogues
+      with 6 utterances.
+  91f33bcf-3c0e-49e6-ae86-28f77e224734: !Template
+    id: 91f33bcf-3c0e-49e6-ae86-28f77e224734
+    jinja:
+      "{% if dialogue_turns.utterance|length==6 %} {{dialogue_turns.utterance[1]}}
+
+
+      {{dialogue_turns.utterance[3]}}
+
+
+      {{dialogue_turns.utterance[4]}}|||{{dialogue_turns.utterance[5]}}{% else %}|||{%
+      endif %}"
+    name: recommend_movie_3
```

Comment:
I don't understand this one:
- 1,3,4,5 ? No 2?
- hard to understand if you don't describe that it is a dialogue

Reply:
The only true dialogue task is `next_utterance`, others are variants which are not exactly dialogue tasks.

Reply:
see comment above, if you remove the history (i.e. the logical connections between utterances, you need to reintroduce it somehow otherwise it looks like a concatenation of random entities)